torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -116,6 +116,38 @@ static const OperatorGeneratorArgs opGenArgs[] = {
           push(stack, a.cpu());
         },
         aliasAnalysisFromSchema()),
+    OperatorGeneratorArgs(
+        TORCH_SELECTIVE_SCHEMA("aten::numpy_T(Tensor(a) self) -> Tensor(a)"),
```

Comment:
could you make this a overload method since numpy_T is already exist in native_functions.yaml. For example, you can take a look at aten::slice.str or aten::slice.list. Our schema matching logic should be able to automatically figure out which one you are using. 
cc: @eellison

Reply:
You mean for this one or for the 4 ops? All the ops are in `aten::` so I assume that the treatment should be the same for all of them.

Also, I do not know how should I write it. I would appreciate it if you could write this bit for me, as I do not know anything about the JIT part really.

Reply:
> could you make this a overload method since numpy_T is already exist in native_functions.yaml

@tugsbayasgalan why do we need an entry for these ops here given that they are already aten functions?

Reply:
I meant something like aten::numpy_T.<something> should work.